bindings/c/local.mk

```diff
@@ -24,7 +24,7 @@ fdb_c_CFLAGS := $(fdbclient_CFLAGS)
 fdb_c_LDFLAGS := $(fdbrpc_LDFLAGS)
 fdb_c_LIBS := lib/libfdbclient.a lib/libfdbrpc.a lib/libflow.a $(FDB_TLS_LIB)
 fdb_c_STATIC_LIBS := $(TLS_LIBS)
-fdb_c_tests_LIBS := -Llib -lfdb_c
+fdb_c_tests_LIBS := -Llib -lfdb_c -lstdc++
```

Comment:
if `fdb_c` needs `libstdc++`, did we make a mistake somewhere, and not link against it there?  I'm concerned that this is spelling troubles for anyone else that would be linking against `libfdb_c` directly.

Reply:
This turned out to be due to LIBSTDCPP_HACK handling, which we've discussed offline.